roles/nova-common/tasks/ceph_integration.yml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
The ceph integration should be on the nova-compute nodes, which is managed by the `nova-data` role. It is inappropriate to put this in the common role.